### DISCUSSION TITLE
Calling sentry from client.

### DISCUSSION THREAD

**Discussion by Fr3akyMurk** [devSparkle/sentry-roblox#10] [Q&A] [answered]
In your docs you clearly have.
"Note: This method cannot be used from a client script. To forward errors from the client, you must use another installation method."

But you never specify how you would use such a method if that is what you want, could you perhaps make a mini guide on it?
Your friendly friend.
- Mark

**Comment by devSparkle** [comment, marked as answer]
Hi @Fr3akyMurk 

With Version 1.0.0, this is no longer an issue; and not part of our documentation. Please take a look at our setup instructions and reach out with any questions you might have!
